### ISSUE TITLE
Support manifest prefixing for ember-asset-loader

### ISSUE THREAD

**Issue by nathanhammond** [rickharrison/broccoli-asset-rev#111]
This work will unlock `broccoli-asset-rev` usage with lazy-loading engines.

https://github.com/trentmwillis/ember-asset-loader/pull/32

**Comment by akashdsouza** [comment]
Is this something that still needs to be supported?

**Comment by trentmwillis** [comment]
@akashdsouza I don't believe there is any work that needs to be done here anymore. Config changes were just needed in ember-engines & ember-asset-loader.

**Comment by akashdsouza** [comment]
OK. Thanks for the update
